### PR TITLE
Feat/138 notebook

### PR DESCRIPTION
**Pull request by yzchyx**
Create a runnable notebook and change PIA-involved files into relative-import fashion.

Co-authored by: Florian @travela

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.analysis.extraPaths": ["./privacy_evaluator"]
+}
```
Comment (travela) [inline .vscode/settings.json:3]: The .vscode directory should probably not be part of the pull request, right?

**Review by travela — APPROVED** [review]
I've just run the notebook in Colab. All imports are working now, very nice @yzchenyuxuan! I'll also push an updated version of the notebook which will include the install command. 

(.vscode folder should be removed?)

File: .vscode/settings.json (hunk 1 above)
Comment (yzchyx) [reply .vscode/settings.json:3]: correct, just remove it!
